README.md

```diff
@@ -46,13 +46,15 @@ By default `babel-preset-amex` will transpile for the "last 2 versions" and "not
     {
       "serverOnly": true,
       "modern": true,
+      "moduleFormat": "esm"
     }
   ]]
 }
 ```
 
 `serverOnly` - Will transpile only for node.
 `modern` - Will transpile for [common browsers](./browserlist.js) n-1.
+`moduleFormat` - Will transpile to ECMAScript module syntax. Any other string will transpile to CommonJS module syntax.
```

Comment:
```suggestion
`moduleFormat` - Will transpile to ECMAScript module syntax. Any string other than `"esm"` will transpile to CommonJS module syntax.
```

Reply:
This makes it clearer. Will make this change!